src/NewCommand.php

```diff
@@ -59,8 +59,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $output->writeln('<info>Crafting application...</info>');
 
```

Comment:
We now get the file name from version here, and pass it to `download` and `extract` methods.